Source/ASMultiplexImageNode.mm

```diff
@@ -806,7 +806,7 @@ - (void)_fetchImageWithIdentifierFromCache:(id)imageIdentifier URL:(NSURL *)imag
   ASDisplayNodeAssertNotNil(completionBlock, @"completionBlock is required");
 
   if (_cache) {
-    [_cache cachedImageWithURL:imageURL callbackQueue:dispatch_get_main_queue() completion:^(id <ASImageContainerProtocol> imageContainer) {
+    [_cache cachedImageWithURL:imageURL callbackQueue:dispatch_get_main_queue() completion:^(id <ASImageContainerProtocol> imageContainer, BOOL isFromMemoryCache) {
```

Comment:
```suggestion
    [_cache cachedImageWithURL:imageURL callbackQueue:dispatch_get_main_queue() completion:^(id <ASImageContainerProtocol> imageContainer, __unused BOOL isFromMemoryCache) {
```

Reply:
This actually wasn't the intention originally. The protocol is in place so any library can provide image downloading and caching either synchronously or asynchronously. It wasn't intended that Texture would know how the cache is backed. I'm guessing some coupling between PINRemoteImage and Texture has crept back in…

I would recommend reporting async / sync?

Reply:
@jparise Done

@garrettmoon Thanks for the context, have updated to use ansync/sync terminology.